docs/architecture/architecture-overview.adoc

```diff
@@ -0,0 +1,260 @@
+
+== MC Functionality
+=== Create a representation of a Mobile App
+The MC UI extension provides the user with the capabilitiy to create a representation of a Mobile App inside their Openshift project.
+
+MC allows the user to select the Mobile category in the Service Catalog and select a representation of an App type to create. Note; an App representation is a set of basic properties for the App. It is not a running process. Currently its a visual represenation of an App. It is a resource that represents your mobile client app as part of the OpenShift UI. It gives the context and information needed to show you relevant information around your particular mobile runtime as well as allowing the setup of different kind of mobile client builds required.
+
+image::images/5.x-ServiceCatalog-MobileApps.png[width=1280, title="Service Catalog - Mobile App Perspective", link="images/5.x-ServiceCatalog-MobileApps.png"]
+
+See below for an example of a Mobile App representation that uses the CRD.
+
+
+.Example of an instantiated CRD for an Android Mobile App Representation
+====
+ $ oc get mobileclients -o=json
+ {
+     "apiVersion": "v1",
+     "items": [
+         {
+             "apiVersion": "mobile.k8s.io/v1alpha1",
+             "kind": "MobileClient",
+             "metadata": {
+                 "clusterName": "",
+                 "creationTimestamp": "2018-01-04T12:35:12Z",
+                 "deletionGracePeriodSeconds": null,
+                 "deletionTimestamp": null,
+                 "labels": {
+                     "icon": "fa-android"
+                 },
+                 "name": "my.app-1515069311",
+                 "namespace": "myproject",
+                 "resourceVersion": "17980",
+                 "selfLink": "/apis/mobile.k8s.io/v1alpha1/namespaces/myproject/mobileclients/my.app-1515069311",
+                 "uid": "b5840229-f14b-11e7-95e5-b242a86257d4"
+             },
+             "spec": {
+                 "apiKey": "3e88bede-0c80-4df1-8cd9-cf9b5f50b771",
+                 "clientType": "android",
+                 "name": "my.app"
+             }
+         }
+     ],
+     "kind": "List",
+     "metadata": {
+         "resourceVersion": "",
+         "selfLink": ""
+     }
+ }
+====
+Further details on creating a custom resource object (instantiation of the CRD) is detailed https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/[here].
+
+=== Provision a Mobile Service
+Another important stage in the Mobile App Dev process is to provision a Mobile value add Service to your project. This provisioning of a Mobile Service into your project is done via the Service Catalog, its not done through the MC. The MC UI extension has provided the Mobile category for the user to find the services which can be used as part of a Mobile solution.
```

Comment:
are we sticking with this terminology?  ```Mobile value add Service``` sounds strange and a bit contrived

Reply:
Thought that I had that changed back to 'Mobile Service'. We are sticking with the term 'Mobile Service'.